CanDevice: /dev/pcan0
CanBaudrate: 1000000
ModId: 13        # module ID
